File: preview/understand/assistant/create-example-1/create-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using System.Collections.Generic;
using Twilio;
using Twilio.Rest.Preview.Understand;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var styleSheet = new Dictionary<string, Object>()
        {
            {"say_voice", "Polly.Joanna"}
        };

        var assistant = AssistantResource.Create(
            friendlyName: "Styled Assistant",
            uniqueName: "styled-assistant",
            styleSheet: styleSheet
        );

        Console.WriteLine(assistant.Sid);
    }
}

File: preview/understand/assistant/create-example-1/create-example-1.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$assistant = $twilio->preview->understand->assistants
                                         ->create(array(
                                                      "friendlyName" => "Styled Assistant",
                                                      "uniqueName" => "styled-assistant",
                                                      "styleSheet" => array(
                                                          "voice" => array(
                                                              "say_voice" => "Polly.Joanna"
                                                          )
                                                      )
                                                  )
                                         );

print($assistant->sid);